rudder-web/src/main/scala/com/normation/rudder/web/components/RuleEditForm.scala

```diff
@@ -227,10 +227,14 @@ class RuleEditForm(
             $$( "#editRuleZone" ).tabs('select', ${tab});"""
           )) &
           JsRaw(s"""
-
             $$("#editRuleZonePortlet").removeClass("nodisplay");
             ${Replace("details", new RuleCompliance(rule, rootRuleCategory).display).toJsCmd};
             scrollToElement("${idToScroll}", ".rudder_col");
+            $$("#editRuleZone").bind( "show", function(event, ui) {
```

Comment:
Ok, i didn't know that you could 'declare' global variables by doing directly "xxx =" without var before so ok !